.github/workflows/publish_latest_vermeer_image.yml

```
@@ -0,0 +1,72 @@
+    runs-on: ubuntu-latest
+    env:
+      REPOSITORY_URL: apache/incubator-hugegraph-computer
+      BRANCH: master
+      IMAGE_URL: hugegraph/vermeer:latest
+      GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+      OWNER: hugegraph
+      REPO: actions
+      LAST_VERMEER_HASH: ${{vars.LAST_VERMEER_HASH}}
+      
+    steps:
+    - name: Set up Docker Buildx
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)